cocotb/scheduler.py

```diff
@@ -454,9 +454,8 @@ def _event_loop(self, trigger):
                 # to unprime the other triggers that didn't fire
                 scheduling_set = set(scheduling)
                 other_triggers = {
-                    t
+                    self._coro2trigger[coro]
                     for coro in scheduling
-                    for t in self._coro2triggers[coro]
                 } - {trigger}
```

Comment:
Looking more closely, this entire thing is dead code now. Do you want me to do another rev to remove it, or leave it for a future PR?

Reply:
Whatever, fix was easy, and this removed even more lines